Gopkg.toml

```diff
@@ -146,13 +146,33 @@
   name = "github.com/multiformats/go-multihash"
   revision = "8be2a682ab9f254311de1375145a2f78a809b07d"
 
+[[override]]
+  name = "github.com/libp2p/go-libp2p-circuit"
+  revision = "eca2b86a1bcf00adba4f71b3654c3898b19ef421"
+
+[[override]]
+  name = "github.com/status-im/go-multiaddr-ethv4"
+  revision = "cbcba3a7c121e29718d9011ffc9e2b78ac6d5c99"
```

Comment:
i think simply pulling latest changes with dep ensure -update github.com/status-im/go-multiaddr-ethv4 is enough.